Makefile

```diff
@@ -97,6 +97,11 @@ else
 	endif
 endif
 
+# if version = "main" then update version to "nightly"
+ifeq ($(VERSION),main)
+	VERSION := nightly
```

Comment:
`nightly` or `main-nightly`?

`nightly`
 * If we tell users "please use nightly" build and they see "night" and "1.19-nightly", they might be confused.
 * If we tell users "please use main-nightly" build they won't find such name.

`main-night`:
* If we tell users "please use main-nightly" , then users will find "main-night" build.

Reply:
I trust the common sense ... and you an just tell "use the 1.19 nigly" if you dont think of main

Reply:
I agree with "common sense" makes sense, but many users might not know the "nightly" concept clearly. 
"main-nightly" doesn't violate common sense IMO 😄  

For naming style 1: If I tell a user to use "nightly", they might need to search "nightly", and then find 1.19-nightly, wrong .... A problem is that `1.19-nightly` might appear before `nightly` in the search result (order by string, eg: dockerhub search).

For naming style 2: I just need to tell the user to use "main-nightly", then the user goes to the download list, search & find "main-nightly", no ambiguity, use it, perfect.

That's my opinion and preference, a little strong but not blocker if most people think `nightly`+`1.19-nightly` is clearer than `main-nightly`+`1.19-nightly` and the word `nightly` wouldn't mislead users.

Reply:
I agree with this logic as well and have changed it to be `main-nightly`.